shared/domain/src/main/java/gov/va/vro/model/bip/BipClaim.java

```diff
@@ -1,8 +1,12 @@
 package gov.va.vro.model.bip;
 
 import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
 
 /** @author warren @Date 11/8/22 */
+@Component
```

Comment:
not a component, no required arguments

Reply:
@wlin2019 all these domain objects here are duplicates of the ones in shared/domain. Can we consolidate?

Reply:
Yes. There are many fields in those objects can be shared. We should consolidate them. But I would suggest to do it later after we know better what data VRO is needed from BIP and what data it can send to BIP.

I created the BIP data objects based on what BIP API asked as input parameters and what data it responded, plus a group of objects for VRO to take responded BIP data. Since we are not quite sure what parameters VRO will ask BIPs and What responded data VRO will use, I created separated objects. Certainly they have some duplication. But I guess we can consolidate them during the integration process as we know more what VRO need from BIP and what VRO can send to BIP. 

Some BIP input parameter fields have data restrictions, e.g., status string for setting RFD status, etc. BIP provides a set of lookup APIs for those fields. At present, I only create two enum for that. We may need more when we know how BIP-MAS-LIGHTHOUSE data will be integrated.